plugins/amongus/Game.cpp

```diff
@@ -97,17 +108,23 @@ const std::string &Game::context(const AmongUsClient_Fields &fields) {
 	return m_context;
 }
 
-const std::string &Game::identity(const AmongUsClient_Fields &fields, const GameData_PlayerInfo_Fields &playerFields) {
+const std::string &Game::identity(const AmongUsClient_Fields &fields, const PlayerControl_Fields &controlFields) {
+	const auto infoFields   = playerInfoFields(controlFields);
+	const auto outfitFields = playerOutfitFields(infoFields);
```

Comment:
Use of `auto` doesn't conform with our coding guidelines

Reply:
Even if part of the same class?

Reply:
yeah - the point is that you can't deduce the type from this line alone which means, you'll have to look it up, which breaks the flow of reading throw the code